### PR TITLE
Rework `Terminal` public methods

### PR DESCRIPTION
**Pull request by dshil**
Fixes: #86

### CODE REVIEW

File: gits/terminal.py [hunk 1]
```diff
@@ -720,3 +704,27 @@ def dumphtml(self):
...
+
+    #
+    # User visible methods.
+    #
+    def generate_html(self, buf):
+        """Determines the terminal commands. Then the commands are executed.
+        Then the html, representing the state of the terminal after the
+        commands executing, is generated. Commands are either the control
+        characters or the escape sequences, describing in on of the
+        configuration files. """
```
Comment (eugulixes) [inline gits/terminal.py]: I liked the whole idea, but the docstring needs to be slightly reworked. First of all, it doesn't tell anything about text which doesn't belong to either escape or control sequences. The text is printed on the screen as is. Then, the docstring doesn't contain any information about the `buf` parameter. I think it's very important to say that `generate_html` takes bytes obtained from a terminal-oriented program. Finally, there must be some information about the result of the routine i.e. HTML.

Comment (eugulixes) [reply gits/terminal.py]:
```python
        """Splits byte buffer ``buf`` taken from a terminal-oriented program 
        into output, escape and control sequences. The output prints on the 
        screen as is. The escape and control sequences are executed, affecting 
        the output. Finally, the routine generates the HTML document which is 
        ready to be printed in a user's browser.
        """
```

**Comment by eugulixes** [comment]
~@dshil, `gits/terminal.py:730:1: W391 blank line at end of file`~

File: gits/terminal.py (hunk 1 above)
Comment (dshil) [reply gits/terminal.py]: I really love this docstring, thx.

**Review by eugulixes — APPROVED** [review]

**Pull request merged**